src/components/screens/settings-screen.js

```diff
@@ -25,6 +25,10 @@ function SettingsScreen(): Node {
         history.push("/main");
     }
 
+    function clickSound() {
```

Comment:
Funkcja odpowiada za reakcję na zdarzenie, według konwencji react-a powinna zaczynać się od "on"

```suggestion
    function onClickSound() {
```